modules/bitgo/src/v2/coinFactory.ts

```diff
@@ -27,6 +27,7 @@ import {
   Eth,
   Eth2,
   FiatEur,
+  FiatGBP,
```

Comment:
nit: any reason it's `GBP` as opposed to `Gbp` like `Eur` and `Usd`?

Reply:
I decided to do it this was since GBP is an abbreviation and not a shorting (like Eur for Euro)